### DISCUSSION TITLE
Plus is encoded as a space on Vercel

### DISCUSSION THREAD

**Discussion by remorses** [vercel/vercel#8632] [General] [answered]
All paths received by an application deployed on Vercel encode + to a space

Repro: https://github.com/remorses/vercel-repro-plus-encoded-as-space

Visit https://vercel-repro-encoded-as-space.vercel.app/plus+is+not+a+space, it will show a space instead of a +

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out @remorses! The team is already looking into this. You can follow along for updates at https://github.com/orgs/vercel/discussions/133#discussioncomment-3699061
